### ISSUE TITLE
error: the following arguments are required: --model-name

### ISSUE THREAD

**Issue by gabormadarasz2117** [nytud/HuLU-evaluate#4]
When I want to train my model to HuLU according to README, the model-name argument does not work.

$ hulu-evaluate --model_name NYTK/PULI-HuBA130M --output_dir /home/gabor/Dokumentumok/Munka/ --train_epochs 2 --train_batch 8

2025-07-11 13:29:49.092543: E external/local_xla/xla/stream_executor/cuda/cuda_dnn.cc:9261] Unable to register cuDNN factory: Attempting to register factory for plugin cuDNN when one has already been registered
2025-07-11 13:29:49.092579: E external/local_xla/xla/stream_executor/cuda/cuda_fft.cc:607] Unable to register cuFFT factory: Attempting to register factory for plugin cuFFT when one has already been registered
2025-07-11 13:29:49.093836: E external/local_xla/xla/stream_executor/cuda/cuda_blas.cc:1515] Unable to register cuBLAS factory: Attempting to register factory for plugin cuBLAS when one has already been registered
2025-07-11 13:29:49.100832: I tensorflow/core/platform/cpu_feature_guard.cc:182] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: AVX2 FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
2025-07-11 13:29:50.133671: W tensorflow/compiler/tf2tensorrt/utils/py_utils.cc:38] TF-TRT Warning: Could not find TensorRT
usage: hulu-evaluate [-h] [--config-file CONFIG_FILE] [--output-dir OUTPUT_DIR] --model-name MODEL_NAME [--tokenizer-name TOKENIZER_NAME] [--train-epochs TRAIN_EPOCHS]
                     [--train-batch TRAIN_BATCH] [--train-lr TRAIN_LR] [--train-warmup TRAIN_WARMUP] [--train-maxlen TRAIN_MAXLEN] [--seed SEED] [--precision PRECISION]
                     [--use-lora USE_LORA] [--lora-r LORA_R] [--lora-alpha LORA_ALPHA] [--lora-dropout LORA_DROPOUT] [--tasks TASKS [TASKS ...]] [--use-fsdp]
                     [--gradient-accumulation-steps GRADIENT_ACCUMULATION_STEPS]
hulu-evaluate: error: the following arguments are required: --model-name
